### DISCUSSION TITLE
how to reduce the spacing beetwen mainmenu items ?

### DISCUSSION THREAD

**Discussion by gourou69** [templaza/astroid-framework#680] [Q&A]
Hello
I'm starting with astro 2 theme and  i'm looking to reduce  spacing betwen menu items can you help ?  
thanks in advance 
jp

**Comment by Chacapamac** [comment]
You can use the **Web Developer Tools** in your Browser to pinpoint the HTML and CSS that command the  menu visual.

**You will see that each individual **menu items** are made this way.**
“ul” menu container
— “li menu item1”
— “li menu item2”
— “li menu item3”
— “li menu item4”
“End ul” menu container

![Screen Shot 2024-05-15 at 10 31 53 AM](https://github.com/templaza/astroid-framework/assets/1907276/b4a8968c-63e7-49ba-919d-ee6d8bb06b53)

If you inspect a “li” the CSS that control the spacing is:
```
ul.nav.astroid-nav li {
  margin: 0 2px;
}
```

You need to override that CSS to what you want

🚀 **Good Day!**
